src/js/actions/AppActions.js

```diff
@@ -35,8 +37,16 @@ const init = () => {
 
         const path = decodeURI(Player.getAudio().src).replace('file://', '');
```

Comment:
I think this is causing the `null` error, the path reverse may not be well formatted, and is not found in the DB